tests/test_caching.py

```diff
@@ -0,0 +1,52 @@
+#    Copyright 2015-2017 ARM Limited
```

Comment:
ah ok, there's even a Google test just after that that checks
https://github.com/ARM-software/trappy/blob/master/tests/test_copyright.py#L48
So probably some files already have Google in their copyrights, I'll double check